SampleApplications/Workshop/Aggregation/Server/AggregationServer.cs

```diff
@@ -68,12 +67,28 @@ protected override MasterNodeManager CreateMasterNodeManager(IServerInternal ser
             bool ownsTypeModel = true;
             ConfiguredEndpointCollection endpoints = configuration.ParseExtension<ConfiguredEndpointCollection>();
 
+            // start the reverse connect host
+            ReverseConnectManager reverseConnectManager = null;
+            if (configuration.ClientConfiguration?.ReverseConnect != null)
+            {
+                var reverseConnect = configuration.ClientConfiguration.ReverseConnect;
+                // start the reverse connection manager
+                reverseConnectManager = new Opc.Ua.Client.ReverseConnectManager();
+                foreach (var endpoint in reverseConnect.ClientEndpoints)
+                {
+                    reverseConnectManager.AddEndpoint(new Uri(endpoint.EndpointUrl));
+                }
+                reverseConnectManager.StartService(configuration);
```

Comment:
>reverseConnectManager.StartService(configuration); [](start = 16, length = 50)

only start if valid endpoints were added, otherwise dispose manager